content/html/concepts/tags/terms/picture/picture.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: '<picture>'
+Description: 'Represents multiple possible image sources to be applied to different devices and screen-sizes.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Images'
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<picture>` tag represents multiple possible image sources to be applied to different devices and screen-sizes. While it has no semantic meaning, the `<picture>` tag will separate its contents from the rest of the document.
+
+## Syntax
+
+```pseudo
+<picture>
+  <!-- zero or more source tags go here -->
+  <!-- followed by one img tag here -->
```

Comment:
```suggestion
  <!-- Zero or more source tags go here -->
  <!-- followed by one img tag here. -->
```

Reply:
Since this is pseudo, I'm thinking that we can disregard using the comment tags

Reply:
You can also add fallback content below where the image goes, which is a nice backup in case the `<img>` link doesn't work for some reason.